core/src/main/java/io/camunda/connector/impl/inbound/correlation/StartEventCorrelationPoint.java

```diff
@@ -50,6 +50,11 @@ public long getProcessDefinitionKey() {
     return processDefinitionKey;
   }
 
+  @Override
+  public String getExecutionId() {
+    return bpmnProcessId + version + processDefinitionKey;
```

Comment:
Would it make more sense to make it smth like this?

```suggestion
    return bpmnProcessId + "-" + version + "-" + processDefinitionKey;
```